aten/src/ATen/native/native_functions.yaml

```diff
@@ -3443,6 +3443,17 @@
   device_check: NoCheck
   device_guard: False
 
+# NOTE [ _reshape_alias ] is meant to be used in the implementation of reshape.
+# They are not user-facing, hence the leading underscore. Please don't use it
+# anywhere else.
+- func: _reshape_alias(Tensor(a) self, int[] size, int[] stride) -> Tensor(a)
+  variants: function, method
+  device_check: NoCheck
+  device_guard: False
+  dispatch:
+    CPU, CUDA, Meta, QuantizedCPU, QuantizedCUDA: _reshape_alias
```

Comment:
If everything should go to the _reshape_alias (that has the same name as the function in the first line, you can skip the dispatch argument altogether.

Reply:
`reshape` is not supported on e.g. sparse tensors, so it's good to list supported backends here to get a better error message when trying to run it with an unsupported backend.

Reply:
In that case it makes sense. Since the comment was only mentioning MKLDNN that is never used, I was proposing this. But if you actually rely on this error, that looks good!